examples/tutorials/api/irfs.py

```diff
@@ -0,0 +1,458 @@
+from gammapy.maps import MapAxes, MapAxis, WcsGeom
+
+######################################################################
+# Inbuilt Gammapy IRFs
+# --------------------
+#
+
+irf_filename = (
+    "$GAMMAPY_DATA/cta-1dc/caldb/data/cta/1dc/bcf/South_z20_50h/irf_file.fits"
+)
+
+aeff = EffectiveAreaTable2D.read(irf_filename, hdu="EFFECTIVE AREA")
+print(aeff)
+
+
+######################################################################
+# We can see that the Effective Area Table is defined in terms of
+# “energy_true” and `offset` from the camera center
+#
+
+# To see the IRF axes binning, eg, offset
+print(aeff.axes["offset"])
+
+# To get the IRF data
+print(aeff.data)
+
+# the aeff is evaluated at a given energy and offset
+print(aeff.evaluate(energy_true=[1, 10] * u.TeV, offset=[0.2, 2.5] * u.deg))
+
+
+# The peek methods give quick looks into the irfs
+aeff.peek()
+
+
+######################################################################
+# Similarly, we can access other IRFs as well
+#
+
+psf = EnergyDependentMultiGaussPSF.read(irf_filename, hdu="Point Spread Function")
+bkg = Background3D.read(irf_filename, hdu="BACKGROUND")
+edisp = EnergyDispersion2D.read(irf_filename, hdu="ENERGY DISPERSION")
+
+print(bkg)
+
+# To evaluate the irfs, pass the values for each axis
+ev = bkg.evaluate(energy=1 * u.TeV, fov_lon=[1, 2] * u.deg, fov_lat=[1, 2] * u.deg)
+print(ev)
+
+# To evaluate the irfs, pass the values for each axis
+ev = edisp.evaluate(energy_true=1 * u.TeV, offset=[0, 1] * u.deg, migra=[1, 1.2])
+print(ev)
+
+edisp.peek()
+
+print(psf)
+
+
+######################################################################
+# The PointSpreadFunction for the CTA DC1 is stored as a combination of 3
+# Gaussians. Other PSFs, like a `PSF_TABLE` and analytic expressions
+# like KING function are also supported. All PSF claases inherit from a
+# common base `PSF` class
+#
+
+print(psf.axes.names)
+
+# To get the containment radius for a fixed fraction at a given position
+print(
+    psf.containment_radius(
+        fraction=0.68, energy_true=1.0 * u.TeV, offset=[0.2, 4.0] * u.deg
+    )
+)
+
+# Alternatively, to get the containment fraction for at a given position
+print(
+    psf.containment(
+        rad=0.05 * u.deg, energy_true=1.0 * u.TeV, offset=[0.2, 4.0] * u.deg
+    )
+)
+
+
+######################################################################
+# Support for Asymmetric IRFs
+# ---------------------------
+#
+
+
+######################################################################
+# While Gammapy does not have inbuilt classes for supporting asymmetric
+# IRFs (except for `Background3D`), custom classes can be created. For
+# this to work correctly with the `MapDatasetMaker`, only variations
+# with `fov_lon` and `fov_lat` can be allowed.
+#
+# The main idea is that the list of required axes should be correctly
+# mentioned in the class definition.
+#
+# Asymmetric PSFs are not correctly supported at present in the data
+# reduction scheme.
```

Comment:
Maybe you can add that one can however create an asymmetric PSF with the current API, as exemplified here... In this way, the user will not be surprised to see that in the tuto